borg/archiver.py

```diff
@@ -1521,7 +1526,7 @@ def main():  # pragma: no cover
         msg = 'Local Exception.\n%s\n%s' % (traceback.format_exc(), sysinfo())
         exit_code = EXIT_ERROR
     except KeyboardInterrupt:
-        msg = 'Keyboard interrupt.\n%s\n%s' % (traceback.format_exc(), sysinfo())
+        msg = 'Keyboard interrupt.'
```

Comment:
we are losing information here, at least in some cases.

there are 2 cases when users press Ctrl-C:
- they decided otherwise and just want to interrupt borg
- borg is somehow hanging and they want to terminate it

In the second case, the traceback is interesting to see where it is hanging.